doc/source/whatsnew/v0.24.2.rst

```diff
@@ -59,6 +59,7 @@ Bug Fixes
 
 - Bug in :meth:`~pandas.core.groupby.GroupBy.transform` where applying a function to a timezone aware column would return a timezone naive result (:issue:`24198`)
 - Bug in :func:`DataFrame.join` when joining on a timezone aware :class:`DatetimeIndex` (:issue:`23931`)
+- Bug in :func:`DataFrame.pivot_table` where use :class:`IntervalIndex` as pivot index would raise TypeError (:issue:`25814`)
```

Comment:
can you use double-backticks on TypeError

Reply:
Should I update the v0.25.0 whatsnew instead of v0.24.2 ?